src/main/java/com/gtnewhorizons/neid/mixins/early/minecraft/MixinExtendedBlockStorage.java

```diff
@@ -0,0 +1,132 @@
+package com.gtnewhorizons.neid.mixins.early.minecraft;
+
+import java.nio.ByteBuffer;
+import java.nio.ShortBuffer;
+
+import net.minecraft.block.Block;
+import net.minecraft.init.Blocks;
+import net.minecraft.world.chunk.storage.ExtendedBlockStorage;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+import com.gtnewhorizons.neid.NEIDConfig;
+import com.gtnewhorizons.neid.mixins.interfaces.IExtendedBlockStorageMixin;
+
+@Mixin(ExtendedBlockStorage.class)
+public class MixinExtendedBlockStorage implements IExtendedBlockStorageMixin {
+
+    @Shadow
+    private int blockRefCount;
+
+    @Shadow
+    private int tickRefCount;
+
+    // TODO: This can be made private after more of the ASM is removed.
```

Comment:
why is that comment here ? the thing is already private, either the comment or the code needs to be changed